geary-papermc-tracking/src/main/kotlin/com/mineinabyss/geary/papermc/tracking/blocks/Block2Prefab.kt

```diff
@@ -0,0 +1,20 @@
+package com.mineinabyss.geary.papermc.tracking.blocks
+
+import com.mineinabyss.geary.prefabs.PrefabKey
+import org.bukkit.block.data.BlockData
+
+class Block2Prefab {
```

Comment:
Similar to EntityTrackers BukkitEntity2Geary, this is used to store the prefabMap and getters/setters for BlockData -> PrefabKey